peps/pep-0448.rst

```diff
@@ -17,8 +17,7 @@ This PEP proposes extended usages of the ``*`` iterable unpacking
 operator and ``**`` dictionary unpacking operators
 to allow unpacking in more positions, an arbitrary number of
 times, and in additional circumstances.  Specifically,
```

Comment:
```suggestion
times, and in function calls and displays.
```

To simplify things further and avoid a hanging fragment, I'd suggest just making the edit like so.